.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-1.22.4.js
```

Comment:
I run Yarn 2 so this just checks in 1.x. That's what most of this diff is in line count.